integration-tests/test/basic-l1-l2-communication.spec.ts

```diff
@@ -49,7 +49,9 @@ describe('Basic L1<>L2 Communication', async () => {
   })
 
   describe('L2 => L1', () => {
-    it('should be able to perform a withdrawal from L2 -> L1', async () => {
+    it('should be able to perform a withdrawal from L2 -> L1', async function() {
+      await useDynamicTimeoutForWithdrawals(this, env)
```

Comment:
`useDynamicTimeoutForWithdrawals` increases the timeout for a withdrawal test based on the challenge period of the target network. Doesn't do anything when testing locally.

Reply:
Ah, `this` binding with `=>` vs `function`